ansible_collections/arista/avd/roles/eos_designs/doc/custom-structured-configuration.md

```diff
@@ -191,3 +190,157 @@ name_server:
   - 10.10.10.11
   - 10.10.10.12
 ```
+## Variables Precedence
```

Comment:
I think this description should be moved up above the structured config data model stuff. So the user understands the problem before trying to solve it.

Reply:
moved on top